### PR TITLE
Rework the python exporters page

### PR DESCRIPTION
**Pull request by svrnm**
Based on #3355 I updated the python exporters documentation to have a similar structure. 

---

**Preview**: https://deploy-preview-3515--opentelemetry.netlify.app/docs/instrumentation/python/exporters/

### CODE REVIEW

File: content/en/docs/instrumentation/python/exporters.md [hunk 1]
```diff
@@ -17,43 +17,190 @@ learn how to setup exporters following the
...
-### Metrics
-
-Use a [`PeriodicExportingMetricReader`][pemr] to periodically print metrics to
-the console. `PeriodicExportingMetricReader` can be configured to export at a
-different interval, change the
-[temporality](/docs/specs/otel/metrics/data-model/#temporality) for each
-instrument kind, or change the default aggregation for each instrument kind.
-
-#### Temporality Presets
+{% alert title="Note" color="note" %}
```
Comment (chalin) [inline content/en/docs/instrumentation/python/exporters.md:203]:
```suggestion
{{% alert title="Note" color="note" %}}
```

File: content/en/docs/instrumentation/python/exporters.md [hunk 2]
```diff
@@ -99,63 +246,90 @@ Setting `OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE` to any other value than
 `CUMULATIVE`, `DELTA` or `LOWMEMORY` will log a warning and set this environment
 variable to `CUMULATIVE`.
 
-```python
-from opentelemetry import metrics
-from opentelemetry.sdk.metrics import MeterProvider
-from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader, ConsoleMetricExporter
-from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+{% /alert %}
```
Comment (chalin) [inline content/en/docs/instrumentation/python/exporters.md:249]:
```suggestion
{{% /alert %}}
```

File: content/en/docs/instrumentation/python/exporters.md [hunk 3]
```diff
@@ -99,63 +246,90 @@ Setting `OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE` to any other value than
...
+Now following the instruction to setup the [OTLP exporters](#otlp-dependencies).
 
-To send data to an OTLP endpoint or the
-[OpenTelemetry Collector](/docs/collector/getting-started/), you'll want to
-configure an OTLP exporter that sends to your endpoint.
+## Prometheus
 
-First, install an OTLP exporter:
+To send your metric data to [Prometheus](https://prometheus.io/), you can either
+[enable Prometheus' OTLP Receiver](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver)
```
Comment (chalin) [inline content/en/docs/instrumentation/python/exporters.md:272]:
```suggestion
[enable OTLP receiver](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver) for Prometheus
```

File: content/en/docs/instrumentation/python/exporters.md [hunk 4]
```diff
@@ -99,63 +246,90 @@ Setting `OTEL_EXPORTER_METRICS_TEMPORALITY_PREFERENCE` to any other value than
...
-    SERVICE_NAME: "your-service-name"
-})
+## Jaeger
 
-reader = PeriodicExportingMetricReader(ConsoleMetricExporter())
-provider = MeterProvider(resource=resource, metric_readers=[reader])
-metrics.set_meter_provider(provider)
+[Jaeger](https://www.jaegertracing.io/) natively supports OTLP to receive trace
+data. You can run Jaeger in a docker container with the UI accessible on port
+16686 and OTLP enabled on ports 4137 and 4138:
```
Comment (chalin) [inline content/en/docs/instrumentation/python/exporters.md:255]: I prefer port numbers _not_ to be in code font, as you have it here. But I wanted to note that this convention isn't used consistently in this page.

**Review by chalin — APPROVED** [review]
A few more copyedits. Otherwise LGTM overall.

**Comment by cartermp** [comment]
@open-telemetry/python-approvers can you take a look?

**Comment by cartermp** [comment]
@open-telemetry/python-approvers ping!

**Comment by svrnm** [comment]
@open-telemetry/python-approvers PTAL!

**Comment by cartermp** [comment]
Going to merge as per practices: https://github.com/open-telemetry/opentelemetry.io/blob/main/CONTRIBUTING.md#approver-and-maintainer-practices

**Pull request merged**